docs/content/src/learn/tutorials/integration-patterns-and-soa/converting-json-to-xml-and-upload-to-ftp/README.md

```diff
@@ -1,4 +1,4 @@
-# Convert JSON to XML and Upload to FTP
+# JSON to XML Transformation and Upload to FTP
```

Comment:
```suggestion
# Transform JSON to XML and Upload to FTP
```

Suggesting a change for this so that the process is clear.

Reply:
The name should comply with the template name here.